features/ui/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -162,14 +162,21 @@ export function SidebarNavigation() {
     <Container isCollapsed={isSidebarCollapsed}>
       <FixedContainer>
         <Header>
-          <Logo
-            src={
-              isSidebarCollapsed
-                ? "/icons/logo-small.svg"
-                : "/icons/logo-large.svg"
-            }
-            alt="logo"
-          />
+          <Logo>
+            <source
+              media={`(max-width: ${theme.breakpoint.desktop})`}
```

Comment:
Btw there's a getter function for accessing the theme variables that you could use here. Not sure if it makes sense here but it would be consistent with the rest of the app. [You can find it here](https://github.com/profydev/prolog-app-daniel-moderiano/blob/main/styles/theme.ts#L265)

Basically you'd write `breakpoint(theme)('desktop')` instead. I created these getter functions because they look much nicer in the styled-components. At the same time they allow us to rearrange the theme object without altering the styled-components everywhere

Reply:
I will definitely have a closer look at all the theming, it's the one part of styled components I haven't done much with before. I had a go using the getter function but I could not for the life of me figure out how to actually *call* it correctly! Will retry this.